deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -127,9 +127,13 @@ spec:
                                   optional for env vars'
                                 type: string
                               divisor:
+                                anyOf:
```

Comment:
other changes looks good, but, why this was changed?
is you using the correct operator-sdk version?

Reply:
I am using operator-sdk version: "v0.17.0". Is it correct?

Reply:
no, we still on 0.15.1;

Reply:
changed operator-sdk version 0.15.1 and regenerate code